app/Http/Controllers/DMEntryController.php

```diff
@@ -3,21 +3,35 @@
 namespace App\Http\Controllers;
 
 use App\Http\Requests\EntryStoreRequest;
+use App\Models\Adventure;
 use App\Models\Entry;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
 
 class DMEntryController extends Controller
 {
     /**
      * @param \Illuminate\Http\Request $request
-     * @return \Illuminate\Http\Response
+     * @return \Inertia\Response
      */
     public function index(Request $request)
     {
-        $entries = Entry::where('type', Entry::TYPE_DM)->get();
+        $entries = Entry::where('type', Entry::TYPE_DM)
+            ->where('user_id', Auth::id())
+            ->with('character', 'adventure')
+            ->get();
 
-        // this should maybe have its own view..
-        // potential fix in the future
-        return view('entry.index', compact('entries'));
+        return Inertia::render('DMEntry/DMEntry', compact('entries'));
+    }
+
+    /**
+     * @param Request $request
+     * @return \Inertia\Response
+     */
+    public function create(Request $request)
+    {
+        $adventures = Adventure::all();
+        return Inertia::render('Entry/Create/DmEntryCreate', compact('adventures'));
```

Comment:
These lines are covered in #196, so if the coverage is above 90% we're probably fine here